src/Components/CreateImageWizardV2/steps/ImageOutput/ImageOutput.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Dispatch, SetStateAction } from 'react';
+
+import {
+  Text,
+  Alert,
+  Bullseye,
+  Form,
+  Spinner,
+  Title,
+} from '@patternfly/react-core';
+
+import ArchSelect from './ArchSelect';
+import CentOSAcknowledgement from './CentOSAcknowledgement';
+import Environment, { EnvironmentStateType } from './Environment';
+import ReleaseSelect from './ReleaseSelect';
+
+import { Distributions } from '../../../../store/imageBuilderApi';
+import DocumentationButton from '../../../sharedComponents/DocumentationButton';
+
+type ImageOutputPropTypes = {
+  release: Distributions;
+  setRelease: Dispatch<SetStateAction<string>>;
```

Comment:
Is it possible to set this to `Dispatch<SetStateAction<Distributions>>` and the arch to `Dispatch<SetStateAction<ArchitectureItem['arch']>>` similar to the `setEnvironment`?